django/db/backends/dummy/base.py

```diff
@@ -83,4 +83,4 @@ def __init__(self, *args, **kwargs):
         self.validation = BaseDatabaseValidation(self)
 
     def is_usable(self):
-        return True
+        return False
```

Comment:
It does -- the ImproperlyConfigured exception can be raised for real databases.
In django.db.backends.base.base.py:

```
    def check_settings(self):
        if self.settings_dict['TIME_ZONE'] is not None:
            if not settings.USE_TZ:
                raise ImproperlyConfigured(
                    "Connection '%s' cannot set TIME_ZONE because USE_TZ is "
                    "False." % self.alias)
            elif self.features.supports_timezones:
                raise ImproperlyConfigured(
                    "Connection '%s' cannot set TIME_ZONE because its engine "
                    "handles time zones conversions natively." % self.alias)
            elif pytz is None:
                raise ImproperlyConfigured(
                    "Connection '%s' cannot set TIME_ZONE because pytz isn't "
                    "installed." % self.alias)
```

So this particular configuration of timezones & pytz would raise ImproperlyConfigured and cause an attempt to validate the migrations to end without raising an error and without validating the migrations.

It looks like maybe the dummy db backend exists just to raise a consistent and informative exception when there is a problem with a database configuration. If that's the intention, then it shouldn't be used in the case of an empty default, because an empty default is valid and shouldn't raise an exception. Or at least not the same exception as other database backends.